lib/app.rb

```diff
@@ -47,4 +52,25 @@ def run
   def exit
     puts 'Thanks for using this app.'
   end
+
+  def add_music_album
+    print 'Add genre: '
+    genre = gets.chomp.to_s
+    print 'Add author: '
+    author = gets.chomp.to_s
+    print 'Add source: '
+    source = gets.chomp.to_s
+    print 'Add label: '
+    label = gets.chomp.to_s
+    print 'Add publish date(yyyy-mm-dd): '
+    publish_date = gets.chomp.to_s
+    print 'Is it on Spotify(true/false): '
+    on_spotify = Boolean(gets.chomp)
+    obj_genre = Genre.new(name: genre)
+    @items[:music_album].push(MusicAlbum.new(genre: obj_genre, author: author, source: source, label: label,
+                                             publish_date: publish_date, on_spotify: on_spotify))
+    # p @items[:music_album]
```

Comment:
Please remove unused comments.

Reply:
@Vgonma thanks for the feedback, line 72 was deleted.